xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererBase.cpp

```diff
@@ -195,6 +209,83 @@ void CRendererBase::Render(CD3DTexture& target, const CRect& sourceRect, const C
       return;
   }
 
+  if (DX::DeviceResources::Get()->Is10BitSwapchain())
+  {
+    if (buf->color_transfer == AVCOL_TRC_SMPTE2084 && buf->primaries == AVCOL_PRI_BT2020)  // HDR10
```

Comment:
once space between `)` and `//` is enough

Reply:
Done